### DISCUSSION TITLE
Installing Monaco on local machine

### DISCUSSION THREAD

**Discussion by mattdengg** [Dynatrace/dynatrace-configuration-as-code#740] [Q&A] [answered]
I'm trying to install Monaco on my local machine, but when i'm following the steps for installation, I keep getting 
monaco: command not found.

 curl -L https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/releases/download/v1.8.3/monaco-darwin-10.16-amd64 -o monaco
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     26      0 --:--:-- --:--:-- --:--:--    26

chmod +x monaco

monaco

bash: monaco: command not found

Does anyone know how to fix this?

**Comment by UnseenWizzard** [comment, marked as answer]
Hi @mattdengg. 
(answering for linux/mac based on the bash error message above)

To run monaco from the folder you've downloaded it to use `./monaco` 

If you want to install monaco system-wide you can do

> (this assumes you monaco is currently in your Downloads folder!)
> sudo mv ~/Downloads/monaco /usr/local/bin/

> if you're in the folder you've downloaded to
> sudo mv monaco /usr/local/bin/

After that you should be able to run it as `monaco` from any folder

See the installation docs:
https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/Get-started/installation

**Reply by mattdengg** [reply]
Thank you so much I have figured out the problem. I realized that the on the https://dynatrace-oss.github.io/dynatrace-monitoring-as-code/Get-started/installation was not up to date with the actual 1.8.3 version from the https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/releases/

I changed command to:
curl -L https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/releases/download/v1.8.3/monaco-darwin-amd64 -o monaco
for the 1.8.3 version and it worked.
